src/e2e-test/java/io.cdap.plugin/actions/ReplicationActions.java

```diff
@@ -55,87 +43,20 @@ public class ReplicationActions {
     public static String updateCondition = PluginPropertyUtils.pluginProp("updateRowCondition");
     public static String updatedValue = PluginPropertyUtils.pluginProp("updatedRow");
 
-    static {
-        SeleniumHelper.getPropertiesLocators(ReplicationLocators.class);
-    }
-    public static void clickNextButton() throws InterruptedException {
-        TimeUnit.SECONDS.sleep(1);
-        ElementHelper.clickOnElement(ReplicationLocators.next);
-    }
-
-    public static void clickOnOraclePlugin() {
-        ElementHelper.clickOnElement(ReplicationLocators.oraclePlugin);
-    }
-
     public static void selectTable() {
         String table = schemaName + "." + tableName;
-        WaitHelper.waitForElementToBeDisplayed(ReplicationLocators.selectTable(table), 300);
-        AssertionHelper.verifyElementDisplayed(ReplicationLocators.selectTable(table));
-        ElementHelper.clickOnElement(ReplicationLocators.selectTable(table));
-    }
-
-    public static void deployPipeline() {
-        ElementHelper.clickOnElement(ReplicationLocators.deployPipeline);
-    }
-
-    public static void startPipeline() {
-        ElementHelper.clickIfDisplayed(ReplicationLocators.start, ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
-    }
-
-    public static void runThePipeline() {
-        startPipeline();
-        WaitHelper.waitForElementToBeDisplayed(ReplicationLocators.running);
-    }
-
-    public static void openAdvanceLogs() {
-        ReplicationLocators.logs.click();
-        parentWindow = SeleniumDriver.getDriver().getWindowHandle();
-        ArrayList<String> tabs = new ArrayList(SeleniumDriver.getDriver().getWindowHandles());
-        SeleniumDriver.getDriver().switchTo().window(tabs.get(tabs.indexOf(parentWindow) + 1));
-        ReplicationLocators.advancedLogs.click();
-    }
-
-    public static void captureRawLog() {
-        try {
-            String rawLogs = getRawLogs();
-            String logsSeparatorMessage = ConstantsUtil.LOGS_SEPARATOR_MESSAGE
-                    .replace("MESSAGE", "DEPLOYED PIPELINE RUNTIME LOGS");
-            BeforeActions.scenario.write(rawLogs);
-            CdfPipelineRunAction.writeRawLogsToFile(BeforeActions.file, logsSeparatorMessage, rawLogs);
-        } catch (Exception e) {
-            BeforeActions.scenario.write("Exception in capturing logs : " + e);
-        }
-    }
-
-    public static String getRawLogs() {
-        CdfPipelineRunAction.viewRawLogs();
-        ArrayList<String> tabs = new ArrayList(SeleniumDriver.getDriver().getWindowHandles());
-        PageHelper.switchToWindow(tabs.indexOf(parentWindow) + 2);
-        String logs = CdfPipelineRunLocators.logsTextbox.getText();
-        Assert.assertNotNull(logs);
-        PageHelper.closeCurrentWindow();
-        return logs;
+        ElementHelper.clickOnElement(CdfPluginPropertiesLocators.selectReplicationTable(table),300);
```

Comment:
nit: wait time can be shorter here, I think 180 seconds should be good enough.what do you think?

Reply:
Yes, changed to 180 seconds.